plugins/inputs/ravendb/README.md

```diff
@@ -0,0 +1,220 @@
+# RavenDB Input Plugin
+
+Reads metrics from RavenDB servers via monitoring endpoints APIs.
+
+Requires RavenDB Server 5.2+.
+
+### Configuration
+
+The following is an example config for RavenDB. **Note:** The client certificate used should have `Operator` permissions on the cluster.
+
+```toml
+## Global tags useful to group other metrics with the RavenDB Node metrics
+[global_tags]
+ cluster = "ECommerce Stage" 
+
```

Comment:
Remove this part and keep only the RavenDB part.

Reply:
done

Reply:
Please mark as resolved.